casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -165,7 +165,7 @@ class TestNode[F[_]](
       Created(block)    = createBlockResult
     } yield block
 
-  def receive(): F[Unit] = tls.receive(p => handle[F](p), kp(().pure[F])).void
+  def receive(): F[Unit] = tls.handleReceive(p => handle[F](p), kp(().pure[F])).void
```

Comment:
I'm not sure about this change. As I understand it - this actually triggers acquiring of messages

Reply:
The `handleReceive` is not actually the function that is doing processing, it's a function that receives a function which will do the processing.